packages/grafana-prometheus/package.json

```diff
@@ -92,6 +134,7 @@
     "react": "18.3.1",
     "react-dom": "18.3.1",
     "react-select-event": "5.5.1",
+    "rimraf": "6.0.1",
```

Comment:
This was missing. It's included in other packages' devDeps. Including it allows us to run the `clean` script from `packages/grafana-prometheus`.